sdk/storage/Azure.Storage.Common/src/Sas/SasQueryParameters.cs

```diff
@@ -128,7 +128,7 @@ public partial class SasQueryParameters
         /// to accept requests.  When specifying a range, note that the range
         /// is inclusive.
         /// </summary>
-        public IPRange IPRange => _ipRange;
+        public SasIPRange SasIPRange => _sasIpRange;
```

Comment:
And I guess here as well.  Anything publicly exposed to the user should just be `IPRange` as the property name.